Contribute/How-to-add-a-new-verion/index.md

```diff
@@ -0,0 +1,181 @@
+
+Now it's time to create the new article. Depending on the article and which version of Umbraco you will be targetting, these are the two most common scenarios:
+
+* The article is targetting an older Umbraco version and you want to create a new version of the article that targets the *current major version*, or
+* You want to target a version of Umbraco that has not been released yet.
+
+Each scenario involves a different set of steps, which are explained below.
+
+:::tip
+Check out the [File Naming Conventions](../File-Naming-Conventions/#target-a-specific-umbraco-version-using-file-naming-conventions) to learn more about how to use file names to target a specific version of Umbraco when creating a new article.
+:::
+
+### Target the current major version
+
+1. Create a copy of the existing article, e.g. `index.md`.
+2. Rename the *old* article to target the version it was written for, e.g. `index-vX.md` - `X` being the *old* version, e.g. 7.
+3. Rename the new article `index.md`.
+
+You will now have (at least) two versions of the same article. One named `index-v7` which is targetting Umbraco 7 and one named `index` which will target the current major version.
+
+:::tip
+If you're unsure which Umbraco version to target the *old* article for, check the `versionFrom` label inside the article, at the very top.
+:::
+
+### Target an upcoming version
+
+1. Create a copy of the existing article, e.g. `index.md`.
+2. Rename the new article based on the Umbraco version that you want to target, e.g. `index-vX.md` - `X` being the next major version, e.g. 9.
+
+You will now have (at least) two versions of the same article. One named `index` which is targetting the current major version, and one named `index-vX` which is targetting the upcoming Umbraco X version.
+
+## Adding labels to the article
+
+You've now created a new version of an article and have used the file naming conventions to target the correct Umbraco version.
+
+The targetted Umbraco version also needs to be added to the **metadata** in the article itself.
+
+1. Open the new article.
+2. Locate the `versionFrom` label in the metadata section at the very top of the article.
+3. Update the version number to match the version you're targetting.
+
+Example:
+
+```none
+---
+versionFrom: 9.0.0
+meta.Title: "Installing Umbraco"
+meta.Description: "Instructions on installing Umbraco: in VS code, via NuGet or on a Mac"
+---
+```
+
+In some cases, you might also want to update the `meta.Title` and `meta.Description` labels.
+
+See the [Umbraco 9 Metadata requirements](#metadata-requirements) section for more details on which metadata to use when verifying material for Umbraco 9.
+
+## Updating the content
+
+With the metadata in place, it is time to start verifying and updating the content in the article.
+
+When updating the content in the article make sure to
+
+* Update any text that is not correct for the new version,
+* Verify and update all code samples and code snippets,
```

Comment:
I think it should be period instead of comma.

Reply:
Good point 🤔 
I was going for a list that is part of a sentence.
So, if you see the entire list, the final one ends the sentence:

```
* Update any text that is not correct for the new version,
* Verify and update all code samples and code snippets,
* Test and verify that any links in the article is targetting the same version, and
* Update any images and screenshots that are/will be outdated.
```

But it would perhaps be better to have separated items in the list 🤔